CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 3.25.1
```

Comment:
Can you just mark this as `# unreleased`? We don't yet know if the next release will be a point release or a minor release.